### DISCUSSION TITLE
noble titles

### DISCUSSION THREAD

**Discussion by mariannevanharten** [FamilySearch/GEDCOM#573] [Q&A] [answered]
I am looking for a gedcom tag to define a noble term, such as countess.
Ok, I know I can define a custom tag, but may be another option is better.

**Comment by tychonievich** [comment, marked as answer]
[g7:INDI-TITL](https://gedcom.io/specifications/FamilySearchGEDCOMv7.html#individual-attributes) is designed for this purpose.

**Reply by mariannevanharten** [reply]
imho, TITL is only for academic and educational titles, not for noble.

**Reply by tychonievich** [reply]
It is defined in the specification as 

> A formal designation used by an individual in connection with positions of royalty or other social status, such as Grand Duke.

This defining text, including the phrase "royalty or other social status" and the example "Grand Duke", was introduced in version 5.4 and has remained unchanged since. I think that your example of "countess" fits that definition.

I agree that "Title" is a word that is also used in English in other contexts, such as my academic title of "Associate Professor" or my friend's job title of "Senior Office Specialist." whether these qualify as the "social status" that GEDCOM defines INDI.TITL to be for is not clear to me, and I expect that some users will put them in an INDI.TITL while others will not.

**Reply by Norwegian-Sardines** [reply]
I agree, the `INDI.TITL` tag is the appropriate place for all titles.  I subscribe to the following set of “Title Type”.

Imperial, royal and noble ranks
Academic degree
Social titles
Title of honor

I generally disagree with placing these titles in the `NAME` tag even though it is generally used in conjunction with their name, for example “King William”, “John Doe esq.”, or “William Cardinal Phillips”.  This is one of the overarching issues that needs to be resolved when a new name structure is created!

**Reply by dthaler** [reply]
As Luther stated, some people use the `NAME` tag and others use the `TITL` tag for:
* things that are typically written before a name like "Rev.", "Dr.", "Hon.", "Prof.", "Gov.", "Pres."
* things that are typically written after a name like "Esq.", "Ph.D.", "FASG"

A Google search for `GEDC "TITL Dr."` will find many example GEDCOM files using that example, and a search for `GEDC "TITL Esq."` will similarly find examples in the wild.

Last year there was a [Webtrees discussion thread](https://www.webtrees.net/index.php/forum/2-open-discussion/38300-show-shortened-scientific-title-dr-rer-nat-dr) debating this same question of `NAME` vs `TITL`.

As a [Family Historian KB article](https://www.fhug.org.uk/kb/kb-article/names-name-parts-and-name-structures/) points out, "Unlike Occupation and Title, Names (and associated prefixes and suffixes) cannot hold dates of validity; you can be recorded as holding an Occupation between two dates, you can be recorded as having a Title between two dates, but you cannot be recorded as having a Name (and associated prefixes and suffixes) between two dates", so people who want to record a date will use `TITL` as a "social status" title.

This question would probably be a good addition to the [Technical FAQ](https://gedcom.io/techfaqs/).

**Reply by dthaler** [reply]
Discussion in GEDCOM Steering Committee:
This would be good to add to the Technical FAQ to give more exposure to the answer and considerations around `INDI`.`TITL`

**Reply by Norwegian-Sardines** [reply]
>As Luther stated, some people use the NAME tag and others use the TITL tag for:
+things that are typically written before a name like "Rev.", "Dr.", "Hon.", "Prof.", "Gov.", "Pres." 
+things that are typically written after a name like "Esq.", "Ph.D.", "FASG"

Titles can also appear in the middle of a name as well as in:  William Cardinal Phillips

**Reply by mariannevanharten** [reply]
That's why I have _TIT1, _TIT2 and _TITL3, for the terms before, in the middle of the name or behind the name.

**Reply by kiwi3685** [reply]
_That's why I have _TIT1, _TIT2 and _TITL3, for the terms before, in the middle of the name or behind the name._

Isn't that what NPFX, SPFX etc are for?

**Reply by Norwegian-Sardines** [reply]
I don’t believe the `NAME` tag should contain any titles in them!  In a list of names for a person why would I want to list, “John Doe”, “John Doe esq.”, “Senator John Doe” because he at birth he was name John Doe, then became a US Senator, and later became a lawyer.  Makes no sense to me!  I’d rather see just John Doe and a time line of events showing when he became and stoped being a senator, then later became a lawyer!

Then maybe he was in the military and changed rank several times. Is the title Major, Sargent, General also appropriate in that list as well?

His name never really change in all that time!

**Reply by kiwi3685** [reply]
I agree, but would go further.
I see the NAME tag as just a concatenation of a set of sub-tags. As such it doesn't belong in GEDCOM at all. It should be determined by the display software to suit the user requirements.

But I see a lot of "display" requirements creeping into GEDCOM7, so is its purpose changing from just reliable data sharing to something else?

**Reply by Norwegian-Sardines** [reply]
> I agree, but would go further. I see the NAME tag as just a concatenation of a set of sub-tags. As such it doesn't belong in GEDCOM at all. It should be determined by the display software to suit the user requirements.
> 
> But I see a lot of "display" requirements creeping into GEDCOM7, so is its purpose changing from just reliable data sharing to something else?

The `NAME` tag is all that is required by GEDCOM, its subtags are not required and in many cases never transmitted.  If the value of the display of a name is dependent on just the subtags, then the order of those subtags is important because some cultures display the surname first, and others display it last?

**Reply by kiwi3685** [reply]
> The `NAME` tag is all that is required by GEDCOM, its subtags are not required and in many cases never transmitted. >

Of course- but it doesn't have to stay that way surely?

>If the value of the display of a name is dependent on just the subtags, then the order of those subtags is important because some cultures display the surname first, and others display it last?>

Exactly. The subtags would need definition about their purpose, like any tag does, so software can interpret their purpose. Thats what any effective data transfer tool needs to do.

**Reply by Norwegian-Sardines** [reply]
> The subtags would need definition about their purpose, like any tag does, so software can interpret their purpose. Thats what any effective data transfer tool needs to do.

This is why I think that the idea of “how a piece of data” needs to be displayed is indicate either in its “data definition” (i.e. GEDCOM7) or in the transmission data (i.e. markdown/html).  The receiving application needs to know what the data means and how it was meant to be used and displayed!
